packages/nulp_elite/src/pages/learnathon/lernVotingList.js

```diff
@@ -79,14 +78,13 @@ const LernVotingList = () => {
           status: ["Live"],
           content_category: selectedCategory,
```

Comment:
@sonaliTekdi
Make sure your MR title follows the standard format. Let me know if you need a reference or clarification on the format.